Actions/AL-Go-Helper.ps1

```diff
@@ -391,13 +392,14 @@ function ReadSettings {
         "templateBranch"                         = ""
         "appDependencyProbingPaths"              = @()
         "githubRunner"                           = "windows-latest"
+        "cacheImageName"                         = "my"
```

Comment:
is "my" the actual value ?

Reply:
Yes - images will be cached using "my-<version>-<country>" - it just needs to be some standard and my is documented in BcContainerHelper as suggested name.